### PR TITLE
civil-2454-add-staff-profile-in-ccd-config

### PR DESCRIPTION
**Pull request by nagaraju-deekonda**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-2454

### Change description ###
To implement work allocation, amendments and additions that need to be made to the CCD configuration file. 
 Additional roles which need to be added are staff roles for the Legal Advisor and Hearing Admin (Listing Officer). 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by suvarnathekkoot — DISMISSED** [review]

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-SDO-nonprod.json [hunk 1]
```diff
@@ -77,5 +77,41 @@
...
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "STANDARD_DIRECTION_ORDER",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "legal-advisor"
```
Comment (Gulam-Hussain) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-SDO-nonprod.json:87]: Is the below authorisation needed? I think you can just put `caseworker-civil-admin` after this line like so:

```
"legal-advisor",
"caseworker-civil-admin"
```

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-SDO-nonprod.json [hunk 2]
```diff
@@ -77,5 +77,41 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "STANDARD_DIRECTION_ORDER",
```
Comment (Gulam-Hussain) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-SDO-nonprod.json:83]: Is this event coming in a later PR, I don't think it currently exists

File: ccd-definition/AuthorisationCaseState/AuthorisationCaseState.json [hunk 3]
```diff
@@ -86,5 +86,43 @@
...
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseStateID": "JUDICIAL_REFERRAL",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "legal-advisor"
```
Comment (Gulam-Hussain) [inline ccd-definition/AuthorisationCaseState/AuthorisationCaseState.json:96]: same as https://github.com/hmcts/civil-ccd-definition/pull/925/files#r903414117

File: ccd-definition/AuthorisationCaseType/AuthorisationCaseType.json [hunk 4]
```diff
@@ -6,5 +6,23 @@
       "caseworker-civil-admin"
     ],
     "CRUD": "CRU"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "UserRoles": [
+      "caseworker-civil-solicitor",
+      "caseworker-civil-admin",
+      "legal-advisor"
```
Comment (Gulam-Hussain) [inline ccd-definition/AuthorisationCaseType/AuthorisationCaseType.json:15]: same as https://github.com/hmcts/civil-ccd-definition/pull/925/files#r903414117

**Comment by akriti-hmcts** [comment]
stale

**Pull request closed without merging**